modules/@apostrophecms/login/index.js

```diff
@@ -151,6 +151,7 @@ module.exports = {
             expireCookie.expires = new Date(0);
             const name = self.apos.modules['@apostrophecms/express'].sessionOptions.name;
             req.res.header('set-cookie', expireCookie.serialize(name, 'deleted'));
+            req.res.cookie(`${self.apos.shortName}.loggedIn`, 'false');
```

Comment:
We might make this configurable later but it's fine to hardcode it this way for now since it's neatly namespaced.